app/tickers/eth_gas/index.ts

```diff
@@ -0,0 +1,41 @@
+import web3 from '../../../config/web3';
+import redis from '../../../config/redis';
+import { enqueueJob } from './queue';
+import { logger } from '../../../utils/logger';
+
+export const gasTicker = async () => {
+
+  const gasPrice = await web3.eth.getGasPrice();
+  const block = await web3.eth.getBlock('latest');
+  const etherValue = web3.utils.fromWei(String(gasPrice * block.gasLimit), 'ether');
+
+  redis.get('ETH_TX', (err, val: string) => {
```

Comment:
@stuartkuentzel Can we change the key to more specific? Cause `ETH_TX` to me looks very ambiguous. Maybe something like `ETH_GAS_PRICE` etc

Reply:
And because this is used in more than 1 places, I think its better to make a variable and then use that so later if we wanted to change we can just change in just one place.

Reply:
Made a commit that changes it to a variable with a value of 'ETH_GAS_PRICE'